local-cli/runIOS/runIOS.js

```diff
@@ -170,11 +170,23 @@ function buildProject(xcodeProject, udid, scheme, configuration = 'Debug', launc
       '-derivedDataPath', 'build',
     ];
     console.log(`Building using "xcodebuild ${xcodebuildArgs.join(' ')}"`);
+    child_process.exec(`sed -i '.orig' "s/const NSUInteger kRCTBundleURLProviderDefaultPort = [0-9]*;/const NSUInteger kRCTBundleURLProviderDefaultPort = ${port};/g" "${__dirname}/../../React/Base/RCTBundleURLProvider.m"`, (err, stdout, stderr) => {
```

Comment:
Can we use a preprocessor macro instead of modifying the source? I think we already set the `RCT_PACKAGER` env var so we could use that, if I remember correctly there's some config in xcode to hook up defines to env vars.